### PR TITLE
All files added

### PR DESCRIPTION
**Pull request by fed1k**

### CODE REVIEW

File: survey_form.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+           </label> <input type="checkbox" name="radio" id="" value="Covid">Covid </label>
+           </label> <input type="checkbox" name="radio" id="" value="SArS">SArs </label>
+        </fieldset>
+
+        </div>
+        <textarea name="text" id="" cols="30" rows="10">
+
+        </textarea>
+        <div id="button">
+            <button id="submit">Submit</button>
```
Comment (fed1k) [inline survey_form.html:53]: 3. You should give `type` of your `button`

File: survey_form.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="survey_form.css">
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id= "title" > "THE NAIROBI HOSPITAL PATIENT DATA FORM" </h1>
```
Comment (fed1k) [inline survey_form.html:12]: It would be better if do not leave whitespaces.

File: survey_form.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+        <fieldset class="list">
+            <input type="radio" name="radio" id="" value="yes">Yes <br>
+             <input type="radio" name="radio" id="" value="no">No <br>
+        </fieldset>
+
+       <fieldset class="list">
+           </label> <input type="checkbox" name="radio" id="" value="Covid">Covid </label>
+           </label> <input type="checkbox" name="radio" id="" value="SArS">SArs </label>
```
Comment (fed1k) [inline survey_form.html:45]: You should delete attributes that you are not using like `id` in this image

File: survey_form.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            <option value="dropdown">Gripp</option>
+            <option value="dropdown">Moustache</option>
+            <option value="dropdown">Headache</option>
+            <option value="dropdown">Toothache</option>
+        </select>
+
+
+        <fieldset class="list">
+            <input type="radio" name="radio" id="" value="yes">Yes <br>
+             <input type="radio" name="radio" id="" value="no">No <br>
```
Comment (fed1k) [inline survey_form.html:40]: You should delete attributes that you are not using like id in this image

File: survey_form.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    <link rel="stylesheet" href="survey_form.css">
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id= "title" > "THE NAIROBI HOSPITAL PATIENT DATA FORM" </h1>
+    
+    <p id="description"> Patient laboratory details </p>
+    
+    <form id="survey-form" encytype= "text/plain" >
```
Comment (fed1k) [inline survey_form.html:16]: It would be better if do not leave whitespaces.

File: survey_form.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="survey_form.css">
```
Comment (fed1k) [inline survey_form.html:7]: `_` this kind of signs might cause linter errors so you should put another name for you CSS file.
Hint : You can replace it with `-`

File: survey_form.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+    
+    
+    
+    
+  
+    
+ 
+</body>
+</html>
```
Comment (fed1k) [inline survey_form.html:67]: You should delete this unnecessary space.

**Review by fed1k — COMMENTED** [review]
1. Linters are not setup
Solution : You should read this article about Linters https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md

**Review by fed1k — COMMENTED** [review]
Your Readme.md file is good but it should be more descriptive

**Review by leolpaz — APPROVED** [review]
Hi @fed1k , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**